osschallenge/views.py

```diff
@@ -115,11 +115,8 @@ def EditProjectView(request, pk):
 def TaskIndexView(request, username=None):
     template_name = 'osschallenge/taskindex.html'
     title = ""
-    try:
-        if rankup_check(request.user.id) is True:
-            return redirect('/rankup/')
-    except ObjectDoesNotExist:
-        pass
```

Comment:
Is it possible to view the TaskIndexView without a logged in user? Or why would there be a `except` for `ObjectDoesNotExist`?

Reply:
Yes it is possible to see the TaskIndexView while you're not logged in. I changed the check.